client/my-sites/upgrades/navigation.jsx

```diff
@@ -44,7 +44,7 @@ var NAV_ITEMS = {
 
 	Email: {
```

Comment:
Good question - I thought about it too, but the path for this nav item is `upgradesPaths.domainManagementEmail`, and we probably don't want to change it too (_then_ we'd have to change the actual URL too ;)). I think just changing the label is the best balance, especially since this is a temporary measure - there are new designs on horizon for this flow :)